mongodbatlas/data_lakes.go

```diff
@@ -0,0 +1,236 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const (
+	dataLakesBasePath = "groups"
+)
+
+// DataLakeService is an interface for interfacing with the Data Lake endpoints of the MongoDB Atlas API.
+// See more: https://docs.mongodb.com/datalake/reference/api/datalakes-api
```

Comment:
```suggestion
// DataLakeService provides access to the data lake related functions in the Atlas API.
//
// See more: https://docs.mongodb.com/datalake/reference/api/datalakes-api
```

please note the extra empty line. When rendering the docs go will ignore new lines (kind of like markdown) so this would be rendered as 
```
DataLakeService is an interface for interfacing with the Data Lake endpoints of the MongoDB Atlas API. See more: https://docs.mongodb.com/datalake/reference/api/datalakes-api
```

I want to slowly fix the existing docs but it may take a while before we get there